docs/pages/desktop-access/getting-started.mdx

```diff
@@ -191,8 +191,9 @@ You can now connect to your Windows desktops from the Teleport Web UI:
 
 ![Connecting to a Windows desktop from the Web UI](../../img/desktop-access/non-ad-connect.png)
 
-## Next Steps
+Teleport can automatically create user on Windows desktop, see the [Automatic User Creation page](./rbac/mdx#automatic-user-creation) to learn more.
```

Comment:
@gravitational/documentation-team I'm not sure if that's the best place to put it, please modify it as you see fit